addon/components/dialog.hbs

```diff
@@ -0,0 +1,42 @@
+{{#if @open}}
```

Comment:
I wonder if we should deviate from the headless-ui naming for arguments like this? To my mind, `@open` is ambiguous - it could be an action name. Perhaps `@isOpen` is better? It would also be symmetrical with [`@isOn` in `<Switch />`](https://github.com/GavinJoyce/ember-headlessui/blob/7cc969213c28c09bc3ae07ab62b2831d0aa17696/addon/components/switch.hbs#L7). What do you think?

Reply:
Yeh, I tossed this same question around in my head a bit. I settled on aligning with the reference implementation as much as I could. I guess the flipside is that `@onSomething` could be seen as representing actions.

It's something I'm super precious about so I'm happy to change to align with what's already in the codebase if you wish mate.

Reply:
And to round this off a little bit we can see that the React implementation exposes an `onClose` prop ([LINK](https://headlessui.dev/react/dialog#component-api)) while the Vue implementation exposes a `close` event ([LINK](https://headlessui.dev/vue/dialog#component-api)) so each implementation is, obviously, built with the patterns of the framework and while `@isSomething` isn't necessarily an Ember pattern, it is expressive and so, if we agree that this is the pattern we want to embrace in this repo I'm all for it.

I'll make the change.

Reply:
Done 👍